src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -238,8 +238,11 @@ class IncidentsListContainer extends React.Component<Props> {
           </PageHeader>
 
           <Alert type="info" icon="icon-circle-info">
-            {t('This page is in beta and currently shows only metric alerts. ')}
-
+            {t('This page is in beta and currently only shows ')}
+            <ExternalLink href="https://docs.sentry.io/workflow/alerts-notifications/alerts/">
+              {t('metric alerts')}
+            </ExternalLink>
+            {t('. ')}
```

Comment:
So this can be done in a better way for our translators (you'll need to import `tct): 

```suggestion
            {tct('This page is in beta and currently only shows [link:metric alerts]. ', {
            link:             <ExternalLink href="https://docs.sentry.io/workflow/alerts-notifications/alerts/" />
            })}
```

Reply:
This way they see the entire sentence when translating

Reply:
Ooh, good suggestion. I didn't like how I add to break up the string there to keep the period unlinked.